src/Core/Commander/Scheduler.js

```diff
@@ -101,7 +101,7 @@ Scheduler.prototype.runCommand = function runCommand(command, queue, executingCo
     });
 };
 
-Scheduler.prototype.execute = function execute(command) {
+Scheduler.prototype.execute = function execute(command, forceRunNow) {
```

Comment:
Rather than adding a boolean which makes code reading harder, I'd prefer a 2nd function.
```js
sheduler.execute(cmd, false);
sheduler.execute(cmd, true);
```
is less clear than:
```js
sheduler.execute(cmd);
sheduler.executeNow(cmd);
```
You could even change the return type for each function.
`execute` would return a Promise, while `executeNow` would return the result (and fail with an error if the requested command cannot be executed in a synchronous fashion).